### ISSUE TITLE
RRD update failed

### ISSUE THREAD

**Issue by lucaderi** [ntop/ntopng#3460]
27/Feb/2020 17:38:03 [second.lua:28] [rrd.lua:387] ERROR: rrd_update_r() [/var/lib/ntopng/-1/rrd/system/-1/cpu_load.rrd][1582821484:0] failed [/var/lib/ntopng/-1/rrd/system/-1/cpu_load.rrd: illegal attempt to update using time 1582821484 when last update time is 1582821484 (minimum one second step)]
27/Feb/2020 17:38:02 [second.lua:28] [rrd.lua:387] ERROR: rrd_update_r() [/var/lib/ntopng/-1/rrd/system/-1/cpu_load.rrd][1582821483:0] failed [could not lock RRD]
27/Feb/2020 17:38:01 [second.lua:28] [rrd.lua:387] ERROR: rrd_update_r() [/var/lib/ntopng/-1/rrd/system/-1/cpu_load.rrd][1582821482:0] failed [could not lock RRD]
27/Feb/2020 17:38:00 [second.lua:28] [rrd.lua:387] ERROR: rrd_update_r() [/var/lib/ntopng/-1/rrd/system/-1/cpu_load.rrd][1582821481:0] failed [could not lock RRD]

**Comment by simonemainardi** [comment]
The cpu is now updated every 5 seconds as of https://github.com/ntop/ntopng/commit/474fe19e073b8f4699ccc0020ec997f9ffcb6c9e

This reduces system pressure and should also fix the error reported:

```
27/Feb/2020 17:38:03 [second.lua:28] [rrd.lua:387] ERROR: rrd_update_r() [/var/lib/ntopng/-1/rrd/system/-1/cpu_load.rrd][1582821484:0] failed [/var/lib/ntopng/-1/rrd/system/-1/cpu_load.rrd: illegal attempt to update using time 1582821484 when last update time is 1582821484 (minimum one second step)]
```

**Comment by lucaderi** [comment]
Issue still present Ubuntu 18.04

Mar 04 19:04:25 pc-deri ntopng[30874]: [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd][1583345065:105:0] failed [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd: illegal attempt to update using time 1583345065 when last update time is 1583345065 (minimum one second step)]
Mar 04 19:04:25 pc-deri ntopng[30874]: 04/Mar/2020 19:04:24 [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd][1583345065:105:0] failed [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd: illegal attempt to update using time 1583345065 when last update time is 1583345065 (minimum one second step)]
Mar 04 19:04:37 pc-deri ntopng[30874]: 04/Mar/2020 19:04:37 [SyslogCollectorInterface.cpp:153] Incoming connection from 127.0.0.1:47271
Mar 04 19:05:00 pc-deri ntopng[30874]: [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/0/rrd/traffic_rxtx.rrd][1583345100:50700:621413] failed [could not lock RRD]
Mar 04 19:05:00 pc-deri ntopng[30874]: 04/Mar/2020 19:05:00 [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/0/rrd/traffic_rxtx.rrd][1583345100:50700:621413] failed [could not lock RRD]
Mar 04 19:05:36 pc-deri ntopng[30874]: [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd][1583345136:889:0] failed [could not lock RRD]
Mar 04 19:05:36 pc-deri ntopng[30874]: 04/Mar/2020 19:05:35 [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd][1583345136:889:0] failed [could not lock RRD]
Mar 04 19:05:45 pc-deri ntopng[30874]: [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/3/rrd/packets_vs_drops.rrd][1583345145:46546:0] failed [could not lock RRD]
Mar 04 19:05:45 pc-deri ntopng[30874]: 04/Mar/2020 19:05:44 [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/3/rrd/packets_vs_drops.rrd][1583345145:46546:0] failed [could not lock RRD]
Mar 04 19:05:53 pc-deri ntopng[30874]: [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd][1583345153:3419:0] failed [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd: illegal attempt to update using time 1583345153 when last update time is 1583345153 (minimum one second step)]
Mar 04 19:05:53 pc-deri ntopng[30874]: 04/Mar/2020 19:05:52 [second.lua:26] [rrd.lua:413] ERROR: rrd_update_r() [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd][1583345153:3419:0] failed [/var/lib/ntopng/2/rrd/packets_vs_drops.rrd: illegal attempt to update using time 1583345153 when last update time is 1583345153 (minimum one second step)]

**Comment by emanuele-f** [comment]
This should be fixed by 1c7dabf2477bca7d39806caf1481427c9c2abd61 and c17ddb486ef2a049ce3134862b012a002d235ecc

**Comment by lucaderi** [comment]
Closed until we can reproduce it
